server/src/lib/data/request-handlers/get-user-id-from-session-or-bail.ts

```diff
@@ -0,0 +1,12 @@
+import type { Request, Response } from "express";
+
+export const getUserIdFromSessionOrBail = (req: Request, res: Response) => {
```

Comment:
Regardless of the above comment, we should also use this function in all the request handlers that already interacted with the `user_id`. Keeping all this logic in the same place makes it easier to make sweeping changes later on.